ci/test_wheel.sh

```diff
@@ -19,6 +19,12 @@ pushd test_binary_generation
 make
 popd
 
+rapids-logger "Check GPU usage"
+nvidia-smi
+
+rapids-logger "Show Numba CUDA library configuration"
+python -c "from numba.cuda.cudadrv.libs import test; test()"
+
```

Comment:
Do we need to do something differently with the wheel tests? It can't find Numba

```python
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'numba'
```

Reply:
Looks like Numba is just not getting installed. Added to PR: https://github.com/rapidsai/pynvjitlink/pull/57